docs/spec.md

```diff
@@ -6312,6 +6331,24 @@ function returns `true`. If `x` is not a tensor, returns `None`.
 tensors and returns `num_results` slices of `x` along the axis `axis`.
 If `x` is not a tensor or `dim(x, axis) % num_results != 0`, returns `None`.
 
+* `convert_with_quantized_type(x: Value, destination_type: Type)
+-> Value` is defined on tensors and returns the converted value of `x` based on
+the `type(x)` and `destination_type` as follows:
+
+```python
+def convert_with_quantized_type(x: Value, destination_type: Type) -> Value:
+  if type(x) == destination_type:
+    return x
+  if is_quantized(type(x)) and is_quantized(destination_type):
+    return quantize(x, destination_type)
```

Comment:
What happens if only one of the types is quantized?

Reply:
Thanks for the feedback! Such a case is now handled.